templates/init/functions/typescript/package.lint.json

```diff
@@ -9,14 +11,16 @@
     "deploy": "firebase deploy --only functions",
     "logs": "firebase functions:log"
   },
-  "main": "lib/index.js",
   "dependencies": {
     "firebase-admin": "~6.0.0",
     "firebase-functions": "^2.1.0"
   },
   "devDependencies": {
-    "tslint": "~5.8.0",
-    "typescript": "~2.8.3"
+    "@firebase/app-types": "^0.3.2",
```

Comment:
Why is this necessary to include?

Reply:
Before I was having troubles with the firebase typings, but I think that might have gotten fixed with the 6.0.0 release.

I'll remove it.